### ISSUE TITLE
Release schedule for 0.21.0 (preliminary)

### ISSUE THREAD

**Issue by laanwj** [bitcoin/bitcoin#18947]
Here is a proposed release schedule for 0.20.0, the next major release of Bitcoin Core. Like for previous major releases I've aimed for a release 6 months after the last (#17432).

2020-10-01 :construction: 
---------------
- Open Transifex translations for 0.21
- Soft translation string freeze (no large or unnecessary string changes until release)
- Finalize and close translations for 0.19

2020-10-15 :construction: 
---------------
- Feature freeze (bug fixes only until release)
- Translation string freeze (no more source language changes until release)

2020-11-01 :construction: 
---------------
- Split off `0.21` branch from `master`
- Start RC cycle, tag and release `0.21.0rc1`
- Start merging for 0.22 on master branch

2020-12-03 :construction: 
---------------
- Release 0.21.0 final (aim)

As 0.20 is not yet released, this schedule is preliminary and might still move (forward).

If any specific dates or timeframes are a problem for you, let me know.

**Comment by andronoob** [comment]
Sorry, I might be too late for the translation string freeze freeze deadline. But there's still a possibly misleading UI text:
https://github.com/bitcoin/bitcoin/blob/b3527fd2e9be5a94b84433ae229cdf0aaa2d3e7d/src/qt/addressbookpage.cpp#L109

I think "Signing" here means "message sign/verify", doesn't it?

**Comment by laanwj** [comment]
> I think "Signing" here means "message sign/verify", doesn't it?

Yes, correct. I too think it's slightly misleading, because obviously without signing there would be no transactions. It refers to "message signing". But on the other hand it's consistent with the other uses of 'signing' in the UI, because signing of transactions happens in the background.

**Comment by MarcoFalke** [comment]
It would be good to leave a note for the translators in transifex

**Comment by andronoob** [comment]
With PSBT a user knows he "signs" transactions. Maybe he will doubt whether non-legacy addresses can be used for purposes like offline-signing (cold storage).

**Comment by laanwj** [comment]
In any case feel free to file a PR to improve that message (in the GUI
repo), whether it will be merged before 0.21 or not, it's good to have.

**Comment by dongcarl** [comment]
Since I don't think we've split off yet, perhaps we should bump the deadline?

**Comment by MarcoFalke** [comment]
It is hard to predict the day of the split-off. Split-off can happen as soon as the majority of the bugfixes in the milestone has been merged or removed from the milestone: https://github.com/bitcoin/bitcoin/milestone/45

Splitting off before the milestone is empty doesn't help because `rc1` can't be tagged either.

**Comment by laanwj** [comment]
It always slips a bit. I suppose it shouldn't be called a deadline but more of a target. Anyhow all help is welcome in reviewing the remaining bugfix PRs!

**Comment by gloriashaw** [comment]
Hi everyone are you looking for a professional binary, forex and Bitcoin broker/manager who will guide and help manage your trade and help you earn meaningful profits all within seven days contact Mr Barry Silbert now for your investment plan. For he has helped me to earn 10,250 USD just with a little investment capital and with the aid of his trading software system that brings forth good trading signals i was able to trade and cash out on time and am still trading with him, if you need his assistance on how to recover your lost investment in bitcoin/binary Contact him now on whatsaap +447508298691. or contact him on his email address Email: Barrysilbert540 @ gmail com [WITH MR BARRY I BELIEVE THERE ARE STILL GOOD INDIVIDUALS WHO STILL HAVE GOOD INTENTION TO OTHERS. ONCE AGAIN THANKS MR BARRY.]

**Comment by MarcoFalke** [comment]
rc2 is now available:

* https://lists.linuxfoundation.org/pipermail/bitcoin-core-dev/2020-November/000094.html

**Comment by estebansanchez32** [comment]
Alguien sabe español?

**Comment by laanwj** [comment]
The rc cycle is taking longer than expected. I've moved the projected release date forward with two months, from 2020-12-03 to 2021-02-03.

**Comment by laanwj** [comment]
`v0.21.0` (final) was just tagged

**Comment by mrkingfoxx** [comment]
> Sorry, I might be too late for the translation string freeze deadline. But there's still a possibly misleading UI text:
> 
> https://github.com/bitcoin/bitcoin/blob/b3527fd2e9be5a94b84433ae229cdf0aaa2d3e7d/src/qt/addressbookpage.cpp#L109
> 
> I think "Signing" here means "message sign/verify", doesn't it?
